src/Media/ImageUploaderService.php

```diff
@@ -41,7 +41,7 @@ public function __construct(
         int $maxFileSize = null
     ) {
         if ($maxFileSize < 0) {
-            throw new ImageSizeError('Max file size should be 0 or bigger.');
+            throw new \InvalidArgumentException('Max file size should be 0 or bigger inside config.yml.');
```

Comment:
I would suggest to make this a `RuntimeException`. Since this is not an argument that the client can change, and then it will be clear in the future that this is something we need to fix on our end. And we can make `RuntimeException` always return a 500 Internal Server Error without a doubt, while `InvalidArgumentException` could also be caused by a bad request.